scripts/execute_code.py

```diff
@@ -46,3 +49,23 @@ def execute_python_file(file):
 
     except Exception as e:
         return f"Error: {str(e)}"
+    
+
+
+def exec_shell(command_line):
+
+    print (f"Executing command '{command_line}' in workspace '{WORKSPACE_FOLDER}'")
+
+    args = command_line.split()
+    base_path = os.getcwd()
+
+    os.chdir(f"{base_path}/{WORKSPACE_FOLDER}")
```

Comment:
Since this uses the current directory, and it could already be in the workspace folder, it can result in errors like 

`SYSTEM:  Command exec_shell returned: Error: [Errno 2] No such file or directory: '/app/auto_gpt_workspace/auto_gpt_workspace'`

Reply:
Should be fixed now.

Reply:
> (...) i have an issue with the command "cd".
> 
>
> Executing command 'cd Auto-GPT; chmod +x auto_gpt.sh' in workspace 'auto_gpt_workspace'
> SYSTEM:  Command exec_shell returned: Error: [Errno 2] No such file or directory: 'cd'
> 
> 
> Everytime it uses the cd command in the shell, it looks like it wont change directory. It can successfully git clone, and everything, but the cd command... i could not make it work.

@tibzejoker cd command works now.

@nponeccop I believe that fixes all open issues, except for GPT3.5 sometimes getting the command name wrong... I believe that is something that can happen with any command.

Reply:
I can confirm that you fixed the cd command !
good job ! 🥳